### PR TITLE
Move to go 1.19, NeoGo 0.102+ and NeoC# 3.6+

### LINKED ISSUE #132: Update neo-go version in `cmd`
We need to upgrade neo-go version that is used in `cmd` for basic neo-bench functionality:

- [ ] get rid of all deprecated methods/functions/fields usages
- [ ] refactor go node configuration templates to match the new config format
- [ ] use Actor/Invoker interfaces where possible

### LINKED ISSUE #134: Update to Go 1.19+
We're using outdated Go in cmd/go.mod, check neo-go issues for required changes.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #132, close #134.

### CODE REVIEW

File: .docker/build/Dockerfile.golang [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-FROM golang:1.17-alpine as builder
+FROM golang:1.19-alpine as builder
 
 WORKDIR /neo-go
 
 # Install deps:
 RUN apk add --no-cache git
 
-ARG REV="v0.99.2"
+ARG REV="890f64007a737d1934adf324e9c3c53da048b214"
```
Comment (roman-khimov) [inline .docker/build/Dockerfile.golang:9]: Released 0.102.0 for now?

File: cmd/internal/prepare.go [hunk 2]
```diff
@@ -197,17 +200,20 @@ func fillChain(ctx context.Context, c *client.Client, proto result.Protocol, sgn
...
+	gasC := gas.NewReader(inv)
 	fs := make([]func() (bool, error), 0, len(opts.Senders)*2)
 	for i := 0; i < len(opts.Senders); i++ {
 		addr := opts.Senders[i].GetScriptHash()
 		fs = append(fs,
 			func() (bool, error) {
-				b, err := c.NEP17BalanceOf(neoHash, addr)
-				return b > 0, err
+				b, err := neoC.BalanceOf(addr)
+				return b.Int64() > 0, err
```
Comment (roman-khimov) [inline cmd/internal/prepare.go:212]: `b.Sign() > 0`?

File: .docker/build/Dockerfile.bench [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Builder layer
-FROM golang:1.17-alpine as builder
+FROM golang:1.19-alpine as builder
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply .docker/build/Dockerfile.bench]: Fixed.

File: .docker/build/Dockerfile.golang (hunk 1 above)
Comment (AnnaShaleva) [reply .docker/build/Dockerfile.golang]: Let's use it.

**Comment by AnnaShaleva** [comment]
Mixed consensus is checked with NeoC# v3.6.0 (+ modules v3.6.1) and NeoGo's 890f64007a737d1934adf324e9c3c53da048b214 (latest master), runs OK.

**Pull request merged**